docs/tips-and-tricks.rst

```diff
@@ -45,3 +45,35 @@ Note that currently it's not possible to pin an app to a commit, so if your
 flatpaks are updated either manually or automatically, the downgraded app will
 be included in the updates. See https://github.com/flatpak/flatpak/issues/3078
 
+Bisecting regressions in application builds
+-------------------------------------------
+
+In case the newest builds of an application introduce regressions, you can use
+``flatpak-bisect`` to discover which commit introduced the regression. It works
+just like ``git bisect``.
```

Comment:
Should we mention it can be found in the `scripts` directory in the flatpak source repo? It looks like at least Debian doesn't install it: https://salsa.debian.org/debian/flatpak/blob/debian/master/debian/flatpak.install

Reply:
oh, I didn't know that. Makes sense. I mentioned this in the newest version of this commit.